### PR TITLE
feat : added cart page

### PR DESCRIPTION
**Pull request by ArjunJunna**
Added the cart page for ecommerce

### CODE REVIEW

File: css/pages/cart-page.css [hunk 1]
```diff
@@ -0,0 +1,208 @@
...
+}
+
+.order__container {
+  width: 30rem;
+  padding: 2rem;
+  background-color: var(--card-bg-color);
+  position: sticky;
+  height: min-content;
+  top: calc(78px + 3rem);
+}
```
Comment (Anurag021) [inline css/pages/cart-page.css:27]: if possible try to use rem here insted px; rest all looks good to me.

**Review by Anurag021 — APPROVED** [review]
Code looks good to me.

File: css/pages/cart-page.css (hunk 1 above)
Comment (ArjunJunna) [reply css/pages/cart-page.css:27]: sure thank you

**Review by vishal804 — APPROVED** [review]

**Comment by vishal804** [comment]
Code Looks Good

**Pull request merged**